package.json

```diff
@@ -61,7 +61,7 @@
   ],
   "dependencies": {
     "@types/pg-pool": "2.0.3",
-    "express-rate-limit": "6.11.1",
+    "express-rate-limit": "7.0.1",
```

Comment:
This should be in devDependencies (for types and testing) *and* peerDependencies (for end users), but the peerDependencies version should be as loose as possible, e.g. `>=6`

Reply:
Alright, this was the reason why I did a semver major change since the `express-rate-limit` dependency changed, but with this suggestion it wouldn't be needed anymore.

Reply:
By the way, since I haven't worked with `peerDependencies` before, I just want to clarify that `express-rate-limit` should be in both `peerDependencies` (>=6.0.0) and `devDependencies` (7.0.1), and that it should be removed from `dependencies`. Is that correct?

Reply:
> express-rate-limit should be in both peerDependencies (>=6.0.0) and devDependencies (7.0.1), and that it should be removed from dependencies. Is that correct?

Yes, that is correct. Here are a couple of examples:

* https://github.com/express-rate-limit/rate-limit-redis/blob/102bbc2765d4f52df4399282d6cb56ac0711daf7/package.json#L54-L70
* https://github.com/express-rate-limit/rate-limit-memcached/blob/310a17c93f33d75e0622170a9e465aabc36e8ac3/package.json#L57-L83